test/bigtest/tests/sort-collection-test.js

```diff
@@ -0,0 +1,209 @@
+    number: 2,
+    date: '2018-12-20T11:58:44.000',
+  },
+  {
+    status: statuses.THIRD_STATUS,
+    number: 3,
+    date: '2017-12-20T11:57:44.000',
+  },
+  {
+    status: statuses.SECOND_STATUS,
+    number: 2,
+    date: '2018-11-21T14:22:23.000',
+  },
+  {
+    status: statuses.THIRD_STATUS,
+    number: 2,
+    date: '2007-12-20T11:57:44.000',
+  },
+  {
+    status: statuses.FIRST_STATUS,
+    number: 1,
+    date: '2018-11-26T12:37:14.000',
+  },
+];
+
+const collectionSorted = [
+  {
+    status: statuses.FIRST_STATUS,
+    number: 3,
+    date: '2018-10-20T23:31:45.000',
+  },
+  {
+    status: statuses.FIRST_STATUS,
+    number: 1,
+    date: '2018-11-26T12:37:14.000',
+  },
+  {
+    status: statuses.FIRST_STATUS,
+    number: 1,
+    date: '2018-11-20T13:59:17.000',
+  },
+  {
+    status: statuses.SECOND_STATUS,
+    number: 3,
+    date: '2018-12-20T12:57:44.000',
+  },
+  {
+    status: statuses.SECOND_STATUS,
+    number: 2,
+    date: '2018-11-21T14:22:23.000',
+  },
+  {
+    status: statuses.THIRD_STATUS,
+    number: 3,
+    date: '2017-12-20T11:57:44.000',
+  },
+  {
+    status: statuses.THIRD_STATUS,
+    number: 2,
+    date: '2018-12-20T11:58:44.000',
+  },
+  {
+    status: statuses.THIRD_STATUS,
+    number: 2,
+    date: '2007-12-20T11:57:44.000',
+  },
+];
+
+describe('sortCollection', () => {
+  describe('throws error if', () => {
+    it('there is no property in collection object', () => {
+      const propertyName = 'number';
+      const invalidCollection = [
+        { number: 134 },
+        { name: 'Name' },
+        { number: 754 },
+      ];
+      let errorMessage = '';
+
+      try {
+        sortCollection(invalidCollection, [propertyName]);
+      } catch ({ message }) {
+        errorMessage = message;
+      }
+
+      expect(errorMessage).to.be.equal(`${propertyName} does not exist`);
+    });
+
+    it('invalid collection argument', () => {
+      const invalidArgument = {};
+      let errorMessage = '';
+
+      try {
+        sortCollection(invalidArgument);
+      } catch ({ message }) {
+        errorMessage = message;
+      }
+
+      expect(errorMessage).to.be.equal('collection parameter must be an array');
+    });
+
+    describe('iteratees argument', () => {
+      it('is not an array', () => {
+        const iteratees = {};
+        let errorMessage = '';
+
+        try {
+          sortCollection([], iteratees);
+        } catch ({ message }) {
+          errorMessage = message;
+        }
+
+        expect(errorMessage).to.be.equal('iteratees.map is not a function');
+      });
+
+      it('has invalid values', () => {
+        const iteratees = [6];
+        let errorMessage = '';
+
+        try {
+          sortCollection([], iteratees);
+        } catch ({ message }) {
+          errorMessage = message;
+        }
+
+        expect(errorMessage).to.be.equal('6 is not valid value');
+      });
+    });
+  });
+});
+
+it('works with empty input', () => {
```

Comment:
Is not test below it is doing the same?

Reply:
It was supposed to have no arguments passed to `sortCollection`. Will fix.